lib/tree-view.coffee

```diff
@@ -95,6 +96,16 @@ class TreeView extends View
 
     @on 'mousedown', '.tree-view-resize-handle', (e) => @resizeStarted(e)
 
+    @on 'dragstart', '.entry', (e) => @onDragStart(e)
+
+    @on 'dragenter', '.entry.directory > .header', (e) => @onDragEnter(e)
+
+    @on 'dragleave', '.entry.directory > .header', (e) => @onDragLeave(e)
+
+    @on 'dragover', '.entry', (e) => @onDragOver(e)
+
+    @on 'drop', '.entry', (e) => @onDrop(e)
+
```

Comment:
Absolutely. I'd also move this to a separate method. I just copied it from the old PR and since nobody mentioned this issue, I just left it as it was.